scss/components/_c--form-radios-checkboxes.scss

```diff
@@ -8,8 +8,9 @@
 //
 $radio-check-border-color-focus:     $form-elements-border-color-focus;
 $radio-check-input-dot-height:       $form-elements-line-height;
-$radio-check-input-padding-vertical: $form-elements-padding-vertical + $form-elements-border-width;
-$radio-check-label-font-weight:      400;
+$radio-check-input-padding-vertical: $spacing--sm;
+$radio-check-input-padding-horizontal: $spacing--sm;
+$radio-check-label-font-weight:      600;
```

Comment:
@adamgparsons do you think we should abstract out our font-weights into variables?